docs/howto-tsg.md

```diff
@@ -143,4 +144,19 @@ To diagnose connectivity issues in `Serverless` mode, the most straight forward
 ## Classic mode troubleshooting
 `Classic` mode is obsoleted and is not encouraged to use. When in this mode, Azure SignalR service uses the connected *Server Connections* to determine if current service is in `default` mode or `serverless` mode. This can lead to some intermediate client connectivity issues because, when there is a sudden drop of all the connected *Server Connection* for example due to network instability, Azure SignalR believes it is now switched to `serverless` mode, and clients connected during this period will never be routed to the hosted app server. Enable [service-side logs](#add_logs_service) and check if there are any clients recorded as `ServerlessModeEntered` if you have hosted app server howevever some clients never reach the app server side. If there is any, [abort these client connections](#TODO_Add_REST_API) and let the clients restart can help.
 
-Troubleshooting `classic` mode connectivity and message delivary issues are similar to [troubleshooting default mode issues](#default_mode_tsg).
\ No newline at end of file
+Troubleshooting `classic` mode connectivity and message delivary issues are similar to [troubleshooting default mode issues](#default_mode_tsg).
+
+<a name="service_health"></a>
+## Service health
+
+You can check the health api for service health.
+
+* Request url: `https://{instance_name}.service.signalr.net/v1/api/health`
+* Response status code:
+  * 200: healthy.
+  * 503: your service is unhealthy.
+    You can:
+    * Wait for auto recover.
+    * Or restart instance.
+    
+    If both not work, please contact us.
```

Comment:
shall we also provide a link to the disaster recovery solution? https://docs.microsoft.com/en-us/azure/azure-signalr/signalr-concept-disaster-recovery